composer.json

```
@@ -41,7 +41,7 @@
         "symfony/stopwatch": "^5.4|^6.2",
         "symfony/validator": "^5.4|^6.2",
```

```suggestion
        "symfony/stopwatch": "^5.4|^6.3",
        "symfony/validator": "^5.4|^6.3",
```
I don't want to rush ahead of time, but sf 6.2 is supported till the end of July. Maybe to bump version to `^6.3`, tho this can be done later on as well

---

thanks for the suggestion! but yeah, better to do that in another PR later